src/data.js

```diff
@@ -16,47 +16,115 @@ export const quizData = {
-          href: 'https://ui.dev/var-let-const/',
-        },
-      ],
+      
     },
     {
-      text: 'What does `typeof` do?',
+      text: 'What is the nickname of the Rotterdam artwork Santa Claus?',
       answers: {
-        a: 'changes the type of a primitive value',
-        b: 'returns a string describing the type of a value',
-        c: 'determines if a value is primitive',
-        d: 'can tell the difference between arrays and objects',
+        a: 'Weihnachtsmann',
+        b: 'The buttplug Gnome',
+        c: 'Papai Noel',
+        d: 'Kris Kringle',
       },
       correct: 'b',
       selected: null,
-      links: [
-        {
-          text: 'javascript.info',
-          href: 'https://javascript.info/types#type-typeof',
-        },
-        {
-          text: 'MDN',
-          href:
-            'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof',
-        },
-      ],
-    },
-    // Add more questions here
+    },
+    {
+      text: 'When a Dutchman is talking about a spreeuw, which bird does he refer to??',
+      answers: {
+        a: 'sparrow',
+        b: 'nutcracker',
+        c: 'jackdaw',
+        d: 'starling',
+      },
+      correct: 'd',
+      selected: null,
+    },
+    {
+      text: 'Which saint is celebrated on November 11th?',
+      answers: {
+        a: 'Sint Michielsgestel',
+        b: 'Sint Nicolaas (Saint Nicholas)',
+        c: 'Sint Maarten (Saint Martin)',
+        d: 'Sint Oedenrode',
+      },
+      correct: 'c',
+      selected: null,
+    },
+    {
+      text: 'What is the Dutch city of light?',
+      answers: {
+        a: 'Maastricht',
+        b: 'Breda',
+        c: 'Eindhoven',
+        d: 'Rotterdam',
+      },
+      correct: 'c',
+      selected: null,
+    },
+    {
+      text: 'Cheese is NOT made in which of these cities?',
+      answers: {
+        a: 'Gouda',
+        b: 'Alkmaar',
+        c: 'Stolwijk',
+        d: 'Leerdam',
+      },
+      correct: 'b',
+      selected: null,
+    },
+    {
+      text: 'Who is the most popular saint in the Netherlands?',
+      answers: {
+        a: 'Saint Valentine',
+        b: 'Saint Juttemis',
+        c: 'Saint Patrick',
+        d: 'Saint Nicholas',
+      },
+      correct: 'd',
+      selected: null,
+    },
+    {
+      text: 'The Netherlands is also known as Holland. What did Holland originally mean?',
+      answers: {
+        a: 'Hollow land',
+        b: 'Woodland',
+        c: 'Running land',
+        d: 'Swampy land',
+      },
+      correct: 'b',
+      selected: null,
+    },
+    {
+      text: 'The Netherlands are always associated with one kind of flower. Which one?',
+      answers: {
+        a: 'Tulips',
+        b: 'Roses',
+        c: 'Daisies',
+        d: 'Hyacinths',
+      },
+      correct: 'a',
+      selected: null,
+    },
+    {
+      text: 'How long have the Dutch been making cheese?',
+      answers: {
+        a: '1000 years old',
```

Comment:
Minor grammatical note: instead of 'xxxx years old' you should just say 'xxxx years'.